client/src/components/displayCard.js

```diff
@@ -0,0 +1,32 @@
+//import { urlencoded } from "express";
+import { useState } from "react";
+
+const DisplayCard = ({ card }) => {
+  //QUALITY OF LIFE CODE
+  // function will make special char appear on page, will pass them through function as string
+  function decodeHtml(html) {
```

Comment:
oo this is interesting! just wondering did you get a chance to try it out with the api image info? what was breaking before this helper function?

Reply:
I am having issues with getting the image from the API, I will definitely meet with a mentor in office hours about this.